source/slang/core.meta.slang

```diff
@@ -1232,6 +1248,22 @@ for (int tt = 0; tt < kTypeCount; ++tt)
     }
 }
 
+sb << "\n\n";
+sb << "// Cast from vector<T,1> to T, which is a scalar type\n";
+
+for (int tt = 0; tt < kBaseTypeCount; ++tt)
+{
+    if(kBaseTypes[tt].tag == BaseType::Void) continue;
+
+    const char* tname = kBaseTypes[tt].name;
+
+    sb << "extension " << tname << "\n";
+    sb << "{\n";
+    sb << "    __implicit_conversion($(kConversionCost_VectorToScalar))\n";
```

Comment:
Should be sb<<"..."<<kConversionCost_cxx<<")";

Reply:
That did it!!
Thank you very much.
I was worried for a while.